src/dispatch/static/dispatch/src/case/CaseSummaryTable.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <v-data-table
+      :headers="headers"
+      :items="items"
+      :loading="loading"
+      :items-per-page="-1"
+      disabled-pagination
+      hide-default-footer
+    >
+      <template v-slot:item.case_priority.name="{ item }">
+        <case-priority :priority="item.case_priority.name" />
+      </template>
+      <template v-slot:item.status="{ item }">
+        <case-status :status="item.status" :id="item.id" />
+      </template>
+      <template v-slot:item.reporter="{ item }">
+        <case-participant :participant="item.reporter" />
```

Comment:
There's no component for case participant. Did you forget to add it and import it?

Reply:
I will remove it.

Reply:
Afaict it looks like cases don't track the reporter, I think that's probably something that should be added.

Reply:
Yes, I agree we should add it.